### ISSUE TITLE
Frequent occurrences of duplicate slugs

### ISSUE THREAD

**Issue by brashrebel** [JacksonCollege/jc-theme-2015#103]
I've spent a good deal of time on the site today and am encountering a large number of pages of various post types which include the "-2" in their URL. This occurs when an entry of a certain type is created when another with the same name already exists. To remedy this, the original needs to be deleted permanently and then the new version needs to be edited to remove the "-2". Here's a couple examples:
- /central-campus-2/
- /academic-deans/faculty-resources-2/
- /food-services/potter-center-cafeteria-2/

And there's plenty more besides.

**Comment by banksashleym** [comment]
I was actually going to ask you about this...I have noticed that a lot of the slugs have a -2 at the end. I did some googling about a week ago and deleted any duplicate pages, emptied the trash and then went and tried to change the slug. In some instances it worked but in some it didn't. For example with /cenetral-campus-2 there is no other page that is central campus, but there is an image called "central campus.jpg" and when I go to /central-campus/ the image shows up. Is this just a matter of renaming the image and then changing the slug on the Central Campus page?
